### ISSUE TITLE
Frontend API functions reference wrong DLL

### ISSUE THREAD

**Issue by YorVeX** [kostya9/NetObsBindings#10]
I've been scratching my head why any frontend API call added with #7 directly leads to OBS crashing, when in a discussion about this someone reminded me that in OBS they are coming from a different DLL (obs-frontend-api.dll). So I checked the source here and indeed the wrong DLL (obs.dll) is referenced for all of these calls, e.g.:

```cs
[DllImport("obs", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
public static extern void* obs_frontend_get_main_window();
```
(from [here](https://github.com/kostya9/NetObsBindings/blob/e47c92963eebb9566f1e04e0f90a1fdc3cf4b39d/NetObsBindings/ObsInterop/ObsFrontendApi.cs))

If you want to test this, a simple call like this should work but currently crashes OBS (process just closes without any further message):
```cs
fixed (byte* profileName = Encoding.UTF8.GetBytes("Test Profile"))
  ObsFrontendApi.obs_frontend_create_profile((sbyte*)profileName);

```

**Comment by kostya9** [comment]
In retrospect, that sounds obvious because the source is in a different folder than libobs 🥲

Fixed that in [0.0.1.26-alpha](https://www.nuget.org/packages/NetObsBindings/0.0.1.26-alpha), added a sample app to this repo where tested that it works

**Comment by YorVeX** [comment]
Thanks, it's working and I could now add a properties dialog to the Tools menu using this:
![image](https://user-images.githubusercontent.com/528974/216828957-5cd49dff-f20f-4808-bb68-4937a3f216e9.png)
